### ISSUE TITLE
FF 58 HTML5 Canvas Global Block

### ISSUE THREAD

**Issue by collinbarrett** [pyllyukko/user.js#345]
FF 58 now supports HTML5 canvas image blocking. It provides a per-site permission pop-up. It'd be great if there was a way for user.js to block this globally with one setting.

![canvas](https://user-images.githubusercontent.com/6483057/32524249-b04eac40-c416-11e7-9315-76d3c65301ef.png)

Source: [gHacks](https://www.ghacks.net/2017/10/28/firefox-58-warns-you-if-sites-use-canvas-image-data/)

**Comment by TomasGutierrezL** [comment]
It's already enabled.

When you set privacy.resistFingerprinting to true it enables canvas blocking.

https://bugzilla.mozilla.org/show_bug.cgi?id=967895#c68

**Comment by collinbarrett** [comment]
Right, but I was wondering if there was a setting to essentially "Don't Allow" globally for all sites, rather than answering the prompt on each site.

**Comment by pyllyukko** [comment]
If you use the relaxed branch, you'll have on option to "Always remember my decision".

**Comment by Atavic** [comment]
about:config __canvas.*__: https://github.com/pyllyukko/user.js/issues/38
